pallets/motion/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+authors = [ "Substrate DevHub <https://github.com/substrate-developer-hub>" ]
+description = "FRAME pallet to wrap council calls providing root origin to the council."
+edition = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+name = "pallet-motion"
+publish = false
+repository = { workspace = true }
+version = "4.0.0-dev"
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[dependencies]
+codec = { workspace = true, default-features = false, features = [
+	"derive",
+] }
+frame-benchmarking = { workspace = true, default-features = false, optional = true }
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+log = { workspace = true, default-features = false }
+pallet-collective = { workspace = true, default-features = false }
+scale-info = { workspace = true, default-features = false, features = [
+	"derive",
+] }
+sp-runtime = { workspace = true, default-features = false }
+sp-std = { workspace = true }
+
+[dev-dependencies]
+pallet-balances = { workspace = true, default-features = false }
+sp-core = { workspace = true, default-features = false }
+sp-io = { workspace = true, default-features = false }
+
+[features]
+default = [ "std" ]
+runtime-benchmarks = [
+	"frame-benchmarking/runtime-benchmarks",
+	"pallet-collective/runtime-benchmarks",
+]
+# Compiles the pallet including `SimpleMajorityOrigin` in its `Config` trait.
+simple-majority = [  ]
```

Comment:
Zepter conflicts with taplo the formatting here, making the CI fail on lint checks.
I believe we have already hitted in the past.

Reply:
Disabled zepter for now to be able to see the CI passing: [c0cc278](https://github.com/r0gue-io/pop-node/pull/448/commits/c0cc27898d78fe048715e2b4039ceecdaba82b15)